openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -133,7 +133,7 @@ <h3 translate>Election result</h3>
         index="$index" heading="{{ 'Ballot' | translate }} {{ $index + 1 }}">
       <div ng-controller="AssignmentPollDetailCtrl">
         <!-- action buttons -->
-        <div class="pull-right" os-perms="assignments.can_manage">
+        <div class="pull-right" os-perms="assignments.can_manage" ng-show="assignment.phase != 2">
```

Comment:
You can use $scope.phases I think: `ng-show="assignment.phase != _.find(phases, {'display_name': 'Finished'}).value`

So we do not have a hard coded value here.